index.html

```diff
@@ -17,10 +17,44 @@
   </head>
   <body>
     <header>
-      <div class="container"></div>
+      <div class="header-background"></div>
+      <div class="container">
+        <div class="row">
+          <div class="column-full">
+            <h1 class="header">Code Journal</h1>
+          </div>
+        </div>
+      </div>
     </header>
     <main class="container">
-      <div data-view="entry-form"></div>
```

Comment:
You need to keep this. Like I said when I intro'd, you should not be deleting any of the code already provided. It was provided for a reason. All of the rest of your content should be in-between these tags.